engines/golint/Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM linterhub/image-go
+LABEL maintainer="linterhub@repometric.com"
+RUN apk update && apk add git
```

Comment:
I suggest to minimize the number of layers, e.g. combine several `Run` statements into one. You can find more information [here](https://docs.docker.com/engine/userguide/eng-image/dockerfile_best-practices/#minimize-the-number-of-layers).